README.md

```diff
@@ -105,19 +118,20 @@ STATIC_URL = '/static/'
 STATICFILES_DIRS = (
   '‘/usr/local/devcycle-server/tour_config/static/’
   )
-  
-Restart the apache server to put all changes into effect. 
+
+Restart the apache server to put all changes into effect.
 
 ```
-/etc/init.d/apache2 reload
+sudo /etc/init.d/apache2 reload
 ```
 
-###Migrate the Database Schema using South
+
+###Migrate the Database Schema using South (Setup The Database)
```

Comment:
Yeah! That's the same thing I was thinking, which is why I put the parenthesis. I just wanted another opinion ;)